examples/tutorials/descriptors_filter.ipynb

```diff
@@ -0,0 +1,304 @@
+    "with WorkGraph(\"Calculation Workgraph\") as wg:\n",
+    "    wg.inputs = inputs\n",
+    "    final_structures = {}\n",
+    "\n",
+    "    for i in range(num_structs):\n",
+    "        structure = StructureData(ase=read(initail_structure, index=i))\n",
+    "\n",
+    "        geomopt_calc = wg.add_task(\n",
+    "            geomoptCalc,\n",
+    "            code=wg.inputs.code,\n",
+    "            model=wg.inputs.model,\n",
+    "            arch=wg.inputs.arch,\n",
+    "            precision=wg.inputs.precision,\n",
+    "            device=wg.inputs.device,\n",
+    "            metadata=wg.inputs.metadata,\n",
+    "            fmax=wg.inputs.fmax,\n",
+    "            opt_cell_lengths=wg.inputs.opt_cell_lengths,\n",
+    "            opt_cell_fully=wg.inputs.opt_cell_fully,\n",
+    "            struct=structure,\n",
+    "        )\n",
+    "\n",
+    "        descriptors_calc = wg.add_task(\n",
+    "            descriptorsCalc,\n",
+    "            code=wg.inputs.code,\n",
+    "            model=wg.inputs.model,\n",
+    "            arch=wg.inputs.arch,\n",
+    "            precision=wg.inputs.precision,\n",
+    "            device=wg.inputs.device,\n",
+    "            metadata=wg.inputs.metadata,\n",
+    "            struct=geomopt_calc.outputs.final_structure,\n",
+    "            calc_per_element=True,\n",
+    "        )\n",
+    "\n",
+    "        final_structures[f\"structs{i}\"] = descriptors_calc.outputs.xyz_output\n",
+    "\n",
+    "    wg.add_task(\n",
+    "        process_and_split_data,\n",
+    "        config_types= Str(\"\"),\n",
+    "        n_samples=Int(num_structs),\n",
+    "        prefix= Str(\"\"),\n",
+    "        scale= Float(1.0e5),\n",
+    "        append_mode= Bool(False),\n",
+    "        trajectory_data= final_structures\n",
+    "        )\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de1905c6",
+   "metadata": {},
+   "source": [
+    "Visualise the WorkGraph"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0d119cad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wg\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b99c2d62",
+   "metadata": {},
+   "source": [
+    "Run the tasks"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5c5fd6df",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wg.run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0db33aa6",
+   "metadata": {},
+   "source": [
+    "We should get a dictionary with filepaths:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fe7291b7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wg.tasks.process_and_split_data.outputs.result"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d2f463f6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file = open(\"test.xyz\")\n",
+    "print(file.read())"
```

Comment:
Do we not need to get this from the outputs?

Either way, we really want to load this with ase, not just the raw contents.